src/platform/sdl/CMakeLists.txt

```diff
@@ -24,6 +24,8 @@ if (NOT SDL2_FOUND AND NOT SDL_FOUND)
 	return()
 endif()
 
+add_definitions(-DBUILD_SDL)
```

Comment:
Indeed it is, see https://cmake.org/cmake/help/v3.0/command/add_definitions.html:

> Adds definitions to the compiler command line for sources in the current directory and below.